src/nr_openai_observability/bedrock.py

```diff
@@ -321,25 +321,45 @@ def build_bedrock_events(response, event_dict, time_delta):
             "model": model,
             "request.model": model,
             "response.model": model,
-            "temperature": temperature,
-            "api_type": None,
+            **compat_fields(["temperature", "request.temperature"], temperature),
+            **compat_fields(["api_type", "response.api_type"], None),
             "vendor": vendor,
             "ingest_source": "PythonSDK",
-            "number_of_messages": len(messages), 
-            "response": messages[-1]['content'][:4095],
+            **compat_fields(
+                ["number_of_messages", "response.number_of_messages"], len(messages)
+            ),
+            "response": messages[-1]["content"][:4095],
             **get_trace_details(),
         }
 
         if stop_reason:
-            summary["finish_reason"] = stop_reason
+            summary.update(
+                compat_fields(
+                    ["finish_reason", "response.choices.finish_reason"], stop_reason
+                )
+            )
         if response_tokens:
-            summary["usage.completion_tokens"] = response_tokens
+            summary.update(
+                compat_fields(
+                    ["usage.completion_tokens", "response.usage.completion_tokens"],
+                    response_tokens,
+                )
+            )
         if tokens:
-            summary["usage.total_tokens"] = tokens
+            summary.update(
+                compat_fields(
+                    ["usage.total_tokens", "response.usage.total_tokens"], tokens
+                )
+            )
         if input_tokens:
-            summary["usage.prompt_tokens"] = input_tokens
+            summary.update(
+                compat_fields(
+                    ["usage.prompt_tokens", "response.usage.prompt_tokens"],
+                    input_tokens,
+                )
+            )
         if max_tokens:
-            summary["max_tokens"] = max_tokens
+            summary.update(["max_tokens", "request.max_tokens"], max_tokens)
```

Comment:
I think this one is missing the `compat_fields` function wrapper.

Reply:
Good catch!